### PR TITLE
[tree widget] Add toggle for hiding unavailable stat vars

### PR DESCRIPTION
**Pull request by chejennifer**
Screen Shot 2021-07-22 at 3 38 18 PM
Screen Shot 2021-07-22 at 3 38 27 PM

### CODE REVIEW

File: static/js/stat_var_hierarchy/stat_var_hierarchy.tsx [hunk 1]
```diff
@@ -74,6 +74,8 @@ interface StatVarHierarchyStateType {
   rootSVGs: StatVarGroupInfo[];
   // Select or de-select a stat var with its path.
   togglePath: (sv: string, path?: string[]) => void;
+  //
```
Comment (beets) [inline static/js/stat_var_hierarchy/stat_var_hierarchy.tsx:77]: add a comment?

**Review by beets — APPROVED** [review]
thanks, and super fast turnaround!

File: static/js/stat_var_hierarchy/stat_var_hierarchy.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/stat_var_hierarchy/stat_var_hierarchy.tsx]: done

**Review by shifucun — APPROVED** [review]
Nice!

**Pull request merged**